examples/scripts/counter.js

```diff
@@ -86,54 +105,89 @@ WoT.produce({
             observable: false,
             readOnly: true,
         },
         lastChange: {
+            title: "Last change",
+            titles: {
+                en: "Last change",
+                de: "Letzte Änderung",
+                it: "Ultima modifica",
+            },
             type: "string",
-            description: "last change of counter value",
+            description: "Last change of counter value",
             descriptions: {
-                en: "last change of counter value",
+                en: "Last change of counter value",
                 de: "Letzte Änderung",
-                it: "ultima modifica del valore",
+                it: "Ultima modifica del valore",
             },
             observable: true,
             readOnly: true,
         },
     },
     actions: {
         increment: {
-            description: "Incrementing counter value",
+            title: "Increment",
+            titles: {
+                en: "Increment",
+                de: "Erhöhen",
+                it: "Incrementa",
+            },
+            description: "Increment counter value",
             descriptions: {
-                en: "Incrementing counter value",
-                de: "Zähler erhöhen",
-                it: "incrementare valore",
+                en: "Increment counter value",
+                de: "Zählerwert erhöhen",
+                it: "Incrementa il valore del contatore",
             },
         },
         decrement: {
+            title: "Decrement",
+            titles: {
+                en: "Decrement",
+                de: "Verringern",
+                it: "Decrementa",
+            },
             description: "Decrementing counter value",
             descriptions: {
                 en: "Decrementing counter value",
-                de: "Zähler verringern",
-                it: "decrementare valore",
+                de: "Zählerwer verringern",
+                it: "Decrementare il valore del contatore",
             },
         },
         reset: {
+            title: "Reset",
+            titles: {
+                en: "Reset",
+                de: "Zurücksetzen",
+                it: "Reset",
+            },
             description: "Resetting counter value",
             descriptions: {
                 en: "Resetting counter value",
-                de: "Zähler resettieren",
-                it: "resettare valore",
+                de: "Zählerwert resettieren",
+                it: "Resettare il valore del contatore",
             },
         },
     },
     events: {
         change: {
-            description: "change event",
+            title: "Changed",
+            titles: {
+                en: "Changed",
+                de: "Geändert",
+                it: "Valore modificato",
+            },
+            description: "Change event",
             descriptions: {
-                en: "change event",
+                en: "Change event",
                 de: "Änderungsnachricht",
```

Comment:
"-nachricht" could be replaced with "-ereignis" to achieve a more literal translation, the current version should also be fine :)